Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Image for building packages in docker
+FROM golang:1.13-alpine as builder
+
+RUN apk --no-cache add ruby ruby-dev ruby-etc make gcc g++ rpm git tar && \
+    gem install --no-user-install --no-document fpm
+
+
+
+FROM builder AS build
+COPY . ./graphite-ch-optimizer
+
+RUN cd graphite-ch-optimizer && \
+    make -e CGO_ENABLED=0 build
+
+
+
+FROM alpine:latest
```

Comment:
when you have a builder image that has already built a static binary (which i would recommend!) then you don't have to bundle a whole operating system. use `FROM scratch`

Reply:
We need tzdata and ssl in case of https connection